package.json

```diff
@@ -89,6 +89,7 @@
     "webpack": "1.13.1"
   },
   "dependencies": {
+    "@okta/i18n": "45.3.0",
```

Comment:
this should be removed, same with the changes to `package-lock.json`. We don't need i18n as a dependency, the contents we need are in `packages` folder

Reply:
👍